docs/managing_deis/upgrading-deis.rst

```diff
@@ -63,6 +63,56 @@ Finally, update ``deisctl`` to the new version and reinstall:
     enable PROXY protocol for ports 80 and 443 on the ELB, add a ``TCP 443:443`` listener, and
     change existing targets and health checks from HTTP to TCP.
 
+
+Graceful Upgrade
+----------------
+
+Alternatively, an experimental feature exists to provide the ability to perform a graceful upgrade. Unlike the
+in-place process above, this process keeps the platforms routers and publishers up during the upgrade process.
+This means that there should only be a maximum of around 1-2 seconds of downtime while the routers boot up. Many
```

Comment:
This is based on the actual availability of the routers - there will almost certainly be at least one functional router up at all times. I originally had some text about configuring LBs to avoid that second or two of downtime, but it ended up being a bit much. Ill re-add something short and generic to explain the concern.